apps/dashboard/src/interpret-vision/__mock_data__/visionData.ts

```diff
@@ -3,39 +3,1114 @@
-    "https://th.bing.com/th/id/R.ee6c3b5dda7aba5c34b372bd8409ac49?rik=K%2bmc9gcjVhhYbQ&riu=http%3a%2f%2fclipart-library.com%2fimages%2fBigAn7qMT.jpg&ehk=SNzySL548vPVf1a8PSz5Gv%2bJXdUyh%2bf2VAPT5oVuLtg%3d&risl=&pid=ImgRaw&r=0",
-    "https://th.bing.com/th/id/R.ee6c3b5dda7aba5c34b372bd8409ac49?rik=K%2bmc9gcjVhhYbQ&riu=http%3a%2f%2fclipart-library.com%2fimages%2fBigAn7qMT.jpg&ehk=SNzySL548vPVf1a8PSz5Gv%2bJXdUyh%2bf2VAPT5oVuLtg%3d&risl=&pid=ImgRaw&r=0",
-    "https://th.bing.com/th/id/R.ee6c3b5dda7aba5c34b372bd8409ac49?rik=K%2bmc9gcjVhhYbQ&riu=http%3a%2f%2fclipart-library.com%2fimages%2fBigAn7qMT.jpg&ehk=SNzySL548vPVf1a8PSz5Gv%2bJXdUyh%2bf2VAPT5oVuLtg%3d&risl=&pid=ImgRaw&r=0",
-    "https://th.bing.com/th/id/R.ee6c3b5dda7aba5c34b372bd8409ac49?rik=K%2bmc9gcjVhhYbQ&riu=http%3a%2f%2fclipart-library.com%2fimages%2fBigAn7qMT.jpg&ehk=SNzySL548vPVf1a8PSz5Gv%2bJXdUyh%2bf2VAPT5oVuLtg%3d&risl=&pid=ImgRaw&r=0",
-    "https://th.bing.com/th/id/R.ee6c3b5dda7aba5c34b372bd8409ac49?rik=K%2bmc9gcjVhhYbQ&riu=http%3a%2f%2fclipart-library.com%2fimages%2fBigAn7qMT.jpg&ehk=SNzySL548vPVf1a8PSz5Gv%2bJXdUyh%2bf2VAPT5oVuLtg%3d&risl=&pid=ImgRaw&r=0",
-    "https://th.bing.com/th/id/R.ee6c3b5dda7aba5c34b372bd8409ac49?rik=K%2bmc9gcjVhhYbQ&riu=http%3a%2f%2fclipart-library.com%2fimages%2fBigAn7qMT.jpg&ehk=SNzySL548vPVf1a8PSz5Gv%2bJXdUyh%2bf2VAPT5oVuLtg%3d&risl=&pid=ImgRaw&r=0",
-    "https://th.bing.com/th/id/R.ee6c3b5dda7aba5c34b372bd8409ac49?rik=K%2bmc9gcjVhhYbQ&riu=http%3a%2f%2fclipart-library.com%2fimages%2fBigAn7qMT.jpg&ehk=SNzySL548vPVf1a8PSz5Gv%2bJXdUyh%2bf2VAPT5oVuLtg%3d&risl=&pid=ImgRaw&r=0",
-    "https://th.bing.com/th/id/R.ee6c3b5dda7aba5c34b372bd8409ac49?rik=K%2bmc9gcjVhhYbQ&riu=http%3a%2f%2fclipart-library.com%2fimages%2fBigAn7qMT.jpg&ehk=SNzySL548vPVf1a8PSz5Gv%2bJXdUyh%2bf2VAPT5oVuLtg%3d&risl=&pid=ImgRaw&r=0",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1",
-    "https://th.bing.com/th/id/OIP.5hk5_K6MD4JmD0Aqvv0P-wHaHa?pid=ImgDet&rs=1"
+  classNames: [
```

Comment:
you can load the file containing the class names and set that instead of hardcoding it here.  That should also save you about 1k lines on the PR.

Reply:
deleted unecessary classnames